enzyme/CMakeLists.txt

```diff
@@ -55,6 +55,9 @@ set(LLVM_DIR "${LLVM_ABSOLUTE_DIR}" CACHE FILEPATH "b" FORCE)
 
 if (EXISTS "${LLVM_DIR}/lib/cmake/llvm/LLVMConfig.cmake")
   set(LLVM_DIR "${LLVM_DIR}/lib/cmake/llvm")
+else()
+  message("LLVM Config: ${LLVM_DIR}/lib/cmake/llvm/LLVMConfig.cmake")
+  message(FATAL_ERROR "LLVM Config not found")
```

Comment:
Can you add some more info (e.g. LLVM Config not found at location LLVM_DIR="${LLVM_DIR}")

Reply:
Looking for the LLVM config is too precise (see incorrectly failing .bc CI).
I updated it to only check if the path exists. This still catches typos.